src/UrlParams.ts

```diff
@@ -103,8 +109,17 @@ export const getUrlParams = (
   hash = window.location.hash
 ): UrlParams => {
   let roomAlias: string | null = null;
+  let password: string | null = null;
+
+  const passwordIndex = hash.indexOf(PASSWORD_STRING);
```

Comment:
Using a question mark here is probably technically fine but seems a bit confusing given the overlap with the query param. Maybe just parse it as a regular query string, so separate fields with '&'?

Reply:
Good point